From acd4324fdd0ad93d40a26e2f396118577865dfaf Mon Sep 17 00:00:00 2001
From: Andoni Morales Alastruey <ylatuya@gmail.com>
Date: Mon, 7 Jan 2019 14:14:03 +0100
Subject: [PATCH 6/6] Fix cross-compilation build

libtool's archive_expsym_cmds doesn't work with windows line
endings and it doesn't detect the .def files as having an EXPORT
entry, adding it twice
---
 win32/xmingw32/libtheoradec-all.def | 116 ++++++++++++++++++------------------
 win32/xmingw32/libtheoraenc-all.def |  34 +++++------
 2 files changed, 75 insertions(+), 75 deletions(-)

diff --git a/win32/xmingw32/libtheoradec-all.def b/win32/xmingw32/libtheoradec-all.def
index 566eeb3..c753562 100644
--- a/win32/xmingw32/libtheoradec-all.def
+++ b/win32/xmingw32/libtheoradec-all.def
@@ -1,58 +1,58 @@
-EXPORTS
-; Old alpha API
-	theora_version_string @ 1
-	theora_version_number @ 2
-
-	theora_decode_header @ 3
-	theora_decode_init @ 4
-	theora_decode_packetin @ 5
-	theora_decode_YUVout @ 6
-
-	theora_control @ 7
-
-	theora_packet_isheader @ 8
-	theora_packet_iskeyframe @ 9
-
-	theora_granule_shift @ 10
-	theora_granule_frame @ 11
-	theora_granule_time @ 12
-
-	theora_info_init @ 13
-	theora_info_clear @ 14
-
-	theora_clear @ 15
-
-	theora_comment_init @ 16
-	theora_comment_add @ 17
-	theora_comment_add_tag @ 18
-	theora_comment_query @ 19
-	theora_comment_query_count @ 20
-	theora_comment_clear @ 21
-
-; New theora-exp API
-	th_version_string @ 22
-	th_version_number @ 23
-
-	th_decode_headerin @ 24
-	th_decode_alloc @ 25
-	th_setup_free @ 26
-	th_decode_ctl @ 27
-	th_decode_packetin @ 28
-	th_decode_ycbcr_out @ 29
-	th_decode_free @ 30
-
-	th_packet_isheader @ 31
-	th_packet_iskeyframe @ 32
-
-	th_granule_frame @ 33
-	th_granule_time @ 34
-
-	th_info_init @ 35
-	th_info_clear @ 36
-
-	th_comment_init @ 37
-	th_comment_add @ 38
-	th_comment_add_tag @ 39
-	th_comment_query @ 40
-	th_comment_query_count @ 41
-	th_comment_clear @ 42
+EXPORTS
+; Old alpha API
+	theora_version_string @ 1
+	theora_version_number @ 2
+
+	theora_decode_header @ 3
+	theora_decode_init @ 4
+	theora_decode_packetin @ 5
+	theora_decode_YUVout @ 6
+
+	theora_control @ 7
+
+	theora_packet_isheader @ 8
+	theora_packet_iskeyframe @ 9
+
+	theora_granule_shift @ 10
+	theora_granule_frame @ 11
+	theora_granule_time @ 12
+
+	theora_info_init @ 13
+	theora_info_clear @ 14
+
+	theora_clear @ 15
+
+	theora_comment_init @ 16
+	theora_comment_add @ 17
+	theora_comment_add_tag @ 18
+	theora_comment_query @ 19
+	theora_comment_query_count @ 20
+	theora_comment_clear @ 21
+
+; New theora-exp API
+	th_version_string @ 22
+	th_version_number @ 23
+
+	th_decode_headerin @ 24
+	th_decode_alloc @ 25
+	th_setup_free @ 26
+	th_decode_ctl @ 27
+	th_decode_packetin @ 28
+	th_decode_ycbcr_out @ 29
+	th_decode_free @ 30
+
+	th_packet_isheader @ 31
+	th_packet_iskeyframe @ 32
+
+	th_granule_frame @ 33
+	th_granule_time @ 34
+
+	th_info_init @ 35
+	th_info_clear @ 36
+
+	th_comment_init @ 37
+	th_comment_add @ 38
+	th_comment_add_tag @ 39
+	th_comment_query @ 40
+	th_comment_query_count @ 41
+	th_comment_clear @ 42
diff --git a/win32/xmingw32/libtheoraenc-all.def b/win32/xmingw32/libtheoraenc-all.def
index 36d2dad..90130fd 100644
--- a/win32/xmingw32/libtheoraenc-all.def
+++ b/win32/xmingw32/libtheoraenc-all.def
@@ -1,17 +1,17 @@
-EXPORTS
-; Old alpha API
-	theora_encode_init @ 1
-	theora_encode_YUVin @ 2
-	theora_encode_packetout @ 3
-	theora_encode_header @ 4
-	theora_encode_comment @ 5
-	theora_encode_tables @ 6
-; New theora-exp API
-	th_encode_alloc @ 7
-	th_encode_ctl @ 8
-	th_encode_flushheader @ 9
-	th_encode_ycbcr_in @ 10
-	th_encode_packetout @ 11
-	th_encode_free @ 12
-	TH_VP31_QUANT_INFO @ 13
-	TH_VP31_HUFF_CODES @ 14
+EXPORTS
+; Old alpha API
+	theora_encode_init @ 1
+	theora_encode_YUVin @ 2
+	theora_encode_packetout @ 3
+	theora_encode_header @ 4
+	theora_encode_comment @ 5
+	theora_encode_tables @ 6
+; New theora-exp API
+	th_encode_alloc @ 7
+	th_encode_ctl @ 8
+	th_encode_flushheader @ 9
+	th_encode_ycbcr_in @ 10
+	th_encode_packetout @ 11
+	th_encode_free @ 12
+	TH_VP31_QUANT_INFO @ 13
+	TH_VP31_HUFF_CODES @ 14
-- 
2.7.4

